cmd/juju/status/status_internal_test.go

```diff
@@ -5376,10 +5378,10 @@ func (s *StatusSuite) TestFormatTabularCAASModelTruncatedVersion(c *gc.C) {
 Model  Controller  Cloud/Region  Version
                                  
 
-App  Version            Status  Scale  Charm  Store  Channel  Rev  OS  Address    Message
-bar  mine/image:0.5               0/1                           0      54.32.1.3  
-baz  image:0.6                    0/1                           0      54.32.1.4  
-foo  image:tag@3046a3d            0/1                           0      54.32.1.2  
+App  Version            Status  Scale  Charm  Store     Channel  Rev  OS  Address    Message
+bar  ch:image:0.5                 0/1         charmhub             0      54.32.1.3  
+baz  image:0.6                    0/1                              0      54.32.1.4  
+foo  image:tag@3046a3d            0/1                              0      54.32.1.2  
```

Comment:
From what I can understand from here:
https://docs.docker.com/engine/reference/commandline/images/

It looks like "image@deadbeef" works or "image:tag" but not "image:tag@digest"

Specifically this content:
```
before (<image-name>[:<tag>], <image id> or <image@digest>) - filter images created before given id or references
since (<image-name>[:<tag>], <image id> or <image@digest>) - filter images created since given id or references
```

Reply:
I tested "image:tag@digest" manually and it worked.
The docker library has methods pull pulling out a tag separate to the digest.